public/app/core/selectors/navModel.ts

```diff
@@ -48,6 +33,26 @@ export const getNavModel = (navIndex: NavIndex, id: string, fallback?: NavModel,
   return getNotFoundModel();
 };
 
+function getSectionRoot(node: NavModelItem): NavModelItem {
+  if (!node.parentItem) {
+    return node;
+  }
+
+  const root = (node.parentItem = { ...node.parentItem });
```

Comment:
**Question:** why do we need to update the `node.parentItem` as well?

Reply:
because I update the children property to set active child